libs/ui-toolkit/tooltip/src/lib/tooltip.stories.ts

```diff
@@ -0,0 +1,75 @@
+import { MatTooltipModule } from '@angular/material/tooltip';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { moduleMetadata, storiesOf } from '@storybook/angular';
+import { text, withKnobs } from '@storybook/addon-knobs';
+
+import { TooltipComponent } from './tooltip.component';
+
+storiesOf('Tooltip', module)
```

Comment:
IMHO, great stories for the component! Can you add one more with knobs implemented too?

Reply:
Thanks! Yeah, I tried implementing one with knobs before pushing, but it didn't work. Remember I mentioned that the tooltip wasn't working when I tried using it with props? :( Not sure if there's a workaround, or if it's worth spending time looking for one.